advisories/github-reviewed/2024/09/GHSA-6q32-hq47-5qq3/GHSA-6q32-hq47-5qq3.json

```diff
@@ -12,10 +12,6 @@
     {
       "type": "CVSS_V3",
       "score": "CVSS:3.1/AV:N/AC:L/PR:L/UI:R/S:U/C:H/I:H/A:N"
-    },
-    {
-      "type": "CVSS_V4",
-      "score": "CVSS:4.0/AV:N/AC:L/AT:N/PR:L/UI:N/VC:H/VI:H/VA:N/SC:N/SI:N/SA:N"
```

Comment:
Reverting this change as I didn't intentionally make it. If it was due to the updated fact that this vulnerability was introduced in v2, please let me know.